examples/ecs/generic_systems.rs

```diff
@@ -1,26 +1,28 @@
-use bevy::prelude::*;
-use bevy::ecs::component::Component;
+use bevy::{prelude::*, ecs::component::Component};
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_system(spawn_entities_on_click::<A>.system())
```

Comment:
I think this needs the second type parameter to work :)

Reply:
@alice-i-cecile do you mean the second parameter type for the function?